paddleseg/models/enet.py

```diff
@@ -0,0 +1,631 @@
+            function; otherwise, PReLU is used. Default: True.
+    """
+    def __init__(self,
+                 channels,
+                 internal_ratio=4,
+                 kernel_size=3,
+                 padding=0,
+                 dilation=1,
+                 asymmetric=False,
+                 dropout_prob=0,
+                 bias=False,
+                 relu=True):
+        super(RegularBottleneck, self).__init__()
+
+        if internal_ratio <= 1 or internal_ratio > channels:
+            raise RuntimeError(
+                "Value out of range. Expected value in the "
+                "interval [1, {0}], got internal_scale={1}.".format(
+                    channels, internal_ratio))
+
+        internal_channels = channels // internal_ratio
+
+        if relu:
+            activation = nn.ReLU
+        else:
+            activation = nn.PReLU
+
+        self.ext_conv1 = nn.Sequential(
+            nn.Conv2D(channels,
+                      internal_channels,
+                      kernel_size=1,
+                      stride=1,
+                      bias_attr=bias), layers.SyncBatchNorm(internal_channels),
+            activation())
+
+        if asymmetric:
+            self.ext_conv2 = nn.Sequential(
+                nn.Conv2D(internal_channels,
+                          internal_channels,
+                          kernel_size=(kernel_size, 1),
+                          stride=1,
+                          padding=(padding, 0),
+                          dilation=dilation,
+                          bias_attr=bias),
+                layers.SyncBatchNorm(internal_channels), activation(),
+                nn.Conv2D(internal_channels,
+                          internal_channels,
+                          kernel_size=(1, kernel_size),
+                          stride=1,
+                          padding=(0, padding),
+                          dilation=dilation,
+                          bias_attr=bias),
+                layers.SyncBatchNorm(internal_channels), activation())
+        else:
+            self.ext_conv2 = nn.Sequential(
+                nn.Conv2D(internal_channels,
+                          internal_channels,
+                          kernel_size=kernel_size,
+                          stride=1,
+                          padding=padding,
+                          dilation=dilation,
+                          bias_attr=bias),
+                layers.SyncBatchNorm(internal_channels), activation())
+
+        self.ext_conv3 = nn.Sequential(
+            nn.Conv2D(internal_channels,
+                      channels,
+                      kernel_size=1,
+                      stride=1,
+                      bias_attr=bias), layers.SyncBatchNorm(channels),
+            activation())
+
+        self.ext_regul = nn.Dropout2D(p=dropout_prob)
+
+        self.out_activation = activation()
+
+    def forward(self, x):
+        # Main branch shortcut
```

Comment:
Remove this annotation plz, the code is clear by itself

Reply:
Done.